diff -ruN c/src/external/downloads/lib3mf-2.4.1/Source/Model/Writer/v100/NMR_ResourceDependencySorter.cpp d/src/external/downloads/lib3mf-2.4.1/Source/Model/Writer/v100/NMR_ResourceDependencySorter.cpp
--- c/src/external/downloads/lib3mf-2.4.1/Source/Model/Writer/v100/NMR_ResourceDependencySorter.cpp
+++ d/src/external/downloads/lib3mf-2.4.1/Source/Model/Writer/v100/NMR_ResourceDependencySorter.cpp
@@ -37,6 +37,8 @@
 #include "Model/Classes/NMR_Model.h"
 #include "Model/Classes/NMR_ModelResource.h"
 
+#include <algorithm>
+
 namespace NMR
 {
 
